### PR TITLE
Fixed blog section entries

### PR DESCRIPTION
**Pull request by ankitrgadiya**
Step one in fixing this [issue](https://github.com/fossasia/gci16.fossasia.org/issues/400)

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -42,12 +42,6 @@
   article: Exploring Google Code In
   article_link: http://abhijeetmanohar.blogspot.in/2016/12/hi-friends-long-time-of-splashing-codes.html
 
-- name: Adarsh Kumar
-  blog: https://abhijeetmanohar.blogspot.com
-  blog_img: adarsh-blog2.png
-  article: What I learned after participating in GCI
-  article_link: https://idealadarsh.wordpress.com/2016/12/08/what-i-learned-after-participating-in-google-code-in/
-
```
Comment (jigyasa-grover) [inline _data/blogs.yml:56]: Why was this removed ?

**Comment by jigyasa-grover** [comment]
Provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

**Comment by ankitrgadiya** [comment]
@jig08 don't merge it yet I'll commit the second step then merge them together

**Comment by ankitrgadiya** [comment]
@jig08 Now images are showing, but all the images we have in blog folder are very different in size and so I'm having trouble fitting them into the box need some help on that.
[Live Link](http://ankitrgadiya.me/gci16.fossasia.org)

**Comment by abishekvashok** [comment]
@ankitrgadiya can you explain why you removed two blogs from the blogs.yml

**Comment by ankitrgadiya** [comment]
@Abhi2424shek  One dwij Jha one was repeated and other one was mixture of 2 already present entries more specifically Adarsh Kumar and Abhijeet Manohar

**Comment by abishekvashok** [comment]
@ankitrgadiya fine. I've added a tag to the PR body please don't remove it, it is needed for danger. I will restart the build now...

File: _data/blogs.yml (hunk 1 above)
Comment (ankitrgadiya) [reply _data/blogs.yml:56]: Check this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L23
and this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L46
and this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L52

Adarsh 2 is like made by mixing the first 2 Adarsh Kumar entry and Abhijeet Manohar Entry

**Comment by oanarosca** [comment]
I think this will also solve #237, so we can close that after merging

**Comment by ankitrgadiya** [comment]
Yep, I've already added default image

**Comment by abishekvashok** [comment]
@ankitrgadiya if you have trouble resizing an image ping me on slack

**Comment by jigyasa-grover** [comment]
Couple of points:

- [ ] Resize images
- [ ] For blog images with white backgrounds, the text isnt visible. Fix it.
- [ ] For `Blog of Ainesh Sootha ` no default image comes up. Fix it too.

**Comment by ankitrgadiya** [comment]
@jig08 already working on them, also Blog Post title is not matching with other section's title so I'm fixing that too.

**Comment by ankitrgadiya** [comment]
Image fixed, text background fixed. I'm not sure why no image showing for Blog of  Ainesh Sootha. I can replace it with default one if you want. Separated Blog from design section. 
@jig08 @Princu7 @Abhi2424shek @oanarosca

**Comment by oanarosca** [comment]
His image isn't showing up because the file name in the .yml file doesn't correspond to the actual image name in the images folder. I'll fix it quickly, no worries. Solve the merge conflicts so we can merge your PR :)

**Comment by ankitrgadiya** [comment]
I just realised that someone actually did something in blog section so How can I fix merge conflicts??

**Comment by abishekvashok** [comment]
@ankitrgadiya solve the merge conflicts or maybe someone @fossasia/gci16-maintainers can resolve it.

**Comment by jigyasa-grover** [comment]
@Abhi2424shek Resolved conflicts, @ankitrgadiya thanks !

**Comment by abishekvashok** [comment]
@jig08 nice work...

**Comment by ankitrgadiya** [comment]
@jig08 @Abhi2424shek @oanarosca thanks for helping me.

**Comment by abishekvashok** [comment]
Always welcome @ankitrgadiya
